src/main/kotlin/no/nav/syfo/veiledernavn/VeilederService.kt

```diff
@@ -30,14 +30,19 @@ class VeilederService(
         val axsysVeiledere = axsysConsumer.getAxsysVeiledere(enhetNr)
         val graphApiVeiledere = graphApiConsumer.getVeiledere(axsysVeiledere)
 
-        return axsysVeiledere.map { axsysVeileder ->
-            graphApiVeiledere.find { it.ident == axsysVeileder.appIdent } ?: noGraphApiVeileder(axsysVeileder)
+        val missingInGraphAPI = mutableListOf<String>()
+        val returnList = axsysVeiledere.map { axsysVeileder ->
+            graphApiVeiledere.find { it.ident == axsysVeileder.appIdent } ?: noGraphApiVeileder(axsysVeileder, missingInGraphAPI)
         }
+        if (missingInGraphAPI.isNotEmpty()) {
+            LOG.warn("Fant ikke navn for ${missingInGraphAPI.size} av ${axsysVeiledere.size} veiledere i graphApi! Feilende identer: ${missingInGraphAPI.joinToString()}")
```

Comment:
Evnt legge til en separator (komma feks) på joinToString for lesbarhetens skyld

Reply:
Det er default for joinToString: listOf("A", "B", "C").joinToString() blir "A, B, C"

Reply:
ah, trodde det var mellomrom jeg :D my bad